Cython/Utility/CythonFunction.c

```diff
@@ -1530,7 +1538,11 @@ __pyx_FusedFunction_call(PyObject *func, PyObject *args, PyObject *kw)
 #else
             PyObject *item = __Pyx_PySequence_ITEM(args, i);  if (unlikely(!item)) goto bad;
 #endif
-            PyTuple_SET_ITEM(new_args, i + 1, item);
+#if CYTHON_COMPILING_IN_LIMITED_API
+        PyTuple_SetItem(new_args, i + 1, item);
+#else
+        PyTuple_SET_ITEM(new_args, i + 1, item);
+#endif
```

Comment:
I ask the same question about incref/decref below where it's much more important due to frequency of use, but asking here more generally: instead of having these conditionals everywhere would it make sense to define a common set of conditional macros somewhere that can be used throughout the code base and would be correctly selected for the limited API or not at compile time?